### DISCUSSION TITLE
gh deployment error

### DISCUSSION THREAD

**Discussion by jumiyoon** [community/community#47012] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! I'm getting an error when trying to deploy my gh page.

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
No such file or directory @ dir_chdir - /github/workspace/docs

and this is my repository: 
https://github.com/jumiyoon/phase-2-final-project

I would really appreciate the help!

**Comment by Anmol-Baranwal** [comment, marked as answer]
Have you test the github page locally? Read [here](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)

**Reply by Anmol-Baranwal** [reply]
Hello! It seems like there might be an issue with the file path for your `style.scss` file. The error message indicates that Jekyll is unable to locate the file.

You may want to double-check the file path and ensure that the file is located in the correct directory. You can also check if the file has the correct file extension (`.scss`) and if it has any syntax errors that might be causing the conversion error.

If you're still having trouble, please provide more details on your project structure, the command you're using to deploy your gh page, and any additional error messages you're seeing. This will help me better understand the issue and provide more specific guidance.

**Reply by Anmol-Baranwal** [reply]
@jumiyoon 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by ghost** [comment]
Hi there! It looks like you're encountering an error with the Jekyll converter while trying to deploy your GitHub page. Specifically, the error message says that the converter is unable to find the file `/github/workspace/docs`.

One possible cause of this error could be that the file or directory in question was accidentally deleted or moved. To fix this, you can try checking if the file or directory is still in the correct location, and if not, restoring it from a backup or creating a new one.

Another possible cause of this error could be a misconfiguration in your Jekyll build process. You may want to check your `_config.yml` file to make sure that your source and destination directories are correctly specified, and that any dependencies or plugins required for your build process are installed and up to date.

If you continue to have issues, it may be helpful to try building and deploying your site locally on your machine first to see if you can reproduce the error and troubleshoot it more easily.
